.github/workflows/ci.yml

```
@@ -0,0 +1,21 @@
+jobs:
+  setup:
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-pnpm-setup.yml@main
+
+  test:
+    needs: setup
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-nx-test.yml@main
+
+  lint:
+    needs: setup
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-nx-lint.yml@main
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)